Main/urls.py

```diff
@@ -14,12 +14,14 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 
 
+
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('game/', include('game.urls')),
```

Comment:
Nie przewidujemy wielu dodatkowych route więc najlepiej byłoby jeżeli path '' prowadziłby od razu do widoku ze scoreboardem. Oznacza to tyle, że zamiast include lepiej zrobić od razu wywołanie odpowiedniego widoku.